{{cookiecutter.repo_slug}}/.github/labels.yml

```diff
@@ -13,10 +13,10 @@
   description: "Enhancement of the code, not introducing new features."
 - name: "refactor"
   color: 1d76db
-  description: "Improvement of existing code, not introducing new features."
+  description: "Updating the code with simpler, easier to understand or more efficient syntax or methods, but not introducing new features."
 - name: "performance"
   color: 1d76db
-  description: "Improving performance, not introducing new features."
+  description: "Improving performance of the project, not introducing new features."
```

Comment:
Conceivably one could interpret "performance" as being about the code (especially for compiled code); since labels already exist for that kind of work, I made clear this was about performance of the project (or software, or application, or however we define what's in the repo) and not the code itself.